### PR TITLE
DOCSP-6530: Misc. CET components

### PR DESCRIPTION
**Pull request by sophstad**
[[JIRA](https://jira.mongodb.org/browse/DOCSP-6530)] [[BI Connector Staging](https://docs-mongodbcom-staging.corp.mongodb.com/bi-connector/sophstad/DOCSP-6530/)] Tested builds as well as single-page preview mode.
- Implement `Cond`, `Superscript`, `Subscript`, `LiteralBlock`, `Topic` components
- Improve handling of CSS classes and ListTables
- Support substitutions defined in `snooty.toml` file
- Simplify image handling: Both `Figure` and `Lightbox` now render an `Image` component
- Fix image rendering in preview mode (was broken by https://github.com/mongodb/snooty-parser/pull/28)
- Remove duplicate components that will be simplified by [DOCSP-6667](https://jira.mongodb.org/browse/DOCSP-6667) (`Binary`, `Data`, `Setting`)
- Run Prettier on preview mode files

### CODE REVIEW

File: src/components/Figure.js [hunk 1]
```diff
@@ -63,23 +42,14 @@ export default class Figure extends Component {
...
+      return <Lightbox nodeData={nodeData} />;
     }
     return (
       <div className="figure" style={{ width: getNestedValue(['options', 'figwidth'], nodeData) || 'auto' }}>
-        <img
-          src={imgData}
-          alt={getNestedValue(['options', 'alt'], nodeData) || imgSrc}
-          width="50%"
-          onLoad={this.handleImageLoaded}
-          ref={this.imgRef}
```
Comment (rayangler) [inline src/components/Figure.js]: Are you able to click on images in a guide to activate the lightbox? I'm trying to click on them in `guides` and they aren't popping up.

File: src/components/Paragraph.js [hunk 2]
```diff
@@ -4,13 +4,13 @@ import ComponentFactory from './ComponentFactory';
...
 const SKIP_P_TAGS = ['caption', 'listItem', 'listTable', 'footnote'];
 
-const Paragraph = ({ nodeData, parentNode, position, ...rest }) => {
+const Paragraph = ({ nodeData, parentNode, ...rest }) => {
   // For paragraph nodes that appear inside certain containers, skip <p> tags and just render their contents
   if (SKIP_P_TAGS.includes(parentNode)) {
     return nodeData.children.map((element, index) => <ComponentFactory {...rest} nodeData={element} key={index} />);
   }
   return (
-    <p className={position}>
```
Comment (rayangler) [inline src/components/Paragraph.js]: Why do we not need the position anymore?

Comment (sophstad) [reply src/components/Paragraph.js]: `position` was used to apply classes like `first` and `last` to an HTML tag, but these classnames can actually be applied to many components beyond paragraphs. The [`CSSWrapper`](https://github.com/mongodb/snooty/pull/114/files#diff-83565a542d6b2aec23ae799755d2674a) component is a much more flexible way to add classes to child components, so we don't need to use `position` anymore.

File: src/components/Figure.js (hunk 1 above)
Comment (sophstad) [reply src/components/Figure.js]: Great catch, thank you! Just pushed the fix for this.

**Pull request merged**